styles/main.scss

```diff
@@ -32,98 +32,142 @@ $font-sizes: (
 }
 
 .footer-icon:hover {
-  color:#4592CF
+  color: #4592cf;
+}
+
+.offset-into-above {
+  position: relative;
+
+  @include media-breakpoint-up(xs) {
+    top: -150px;
+  }
+
+  @include media-breakpoint-up(md) {
+    top: -300px;
+  }
+}
+
+// Home page classes
+#home {
+  @include media-breakpoint-up(xs) {
```

Comment:
I would probably try to work these with flexbox first, since it's just pixel length.  You can set the max-height at each breakpoint and let the flexbox grow until that size.